src/vuestic-theme/vuestic-sass/global/ContentClass.demo.vue

```diff
@@ -0,0 +1,175 @@
+<template>
```

Comment:
We need some kind of presentation for this.

Perhaps, you can put it to typography page with a note as follows:

If you have unstyled html and want it to look pretty - just wrap it into `.context` class.
We support the following tags: [list tags].
[Then throw throw various .context tags on the page similarly to other typography presentation]